lib/importers/licence_transaction/licence_importer.rb

```diff
@@ -31,6 +31,11 @@ def call
             next
           end
 
+          if new_licence_already_imported?(new_licence.base_path)
+            puts "Skipping as licence: #{new_licence.base_path} is already imported"
+            next
+          end
+
```

Comment:
This is nice!

Very minor, but I think there's a typo in the commit message. I think this should say option 3:

> I also didn't choose option 2 as this would require multiple editions to be
published / change history that we don't want or need, as it would be
better to have a fresh edition on import.

Reply:
Good spot!